MaxText/pyconfig.py

```diff
@@ -147,6 +147,9 @@ def validate_model_name(s: str) -> bool:
       "llama3-70b",
       "mistral-7b",
       "mixtral-8x7b",
+      "gemma2-2b",
+      "gemma2-9b",
+      "gemma2-27b"
```

Comment:
Can you move it below gemma-* ?
Also there is gemma2-9B in the list.

Reply:
updated